### PR TITLE
Improve database_types docs

### PR DESCRIPTION
**Pull request by mglaman**
Slight adjustments to the database types documentation.

### CODE REVIEW

File: docs/users/extend/database_types.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ## Database Server Types
 
-ddev can currently support most versions of MariaDB and MySQL database servers. You can configure the type that you want using the .ddev/config.yaml file:
+ddev can currently support most versions of MariaDB and MySQL database servers. You can configure the type that you want using the `.ddev/config.yaml` file:
```
Comment (rfay) [inline docs/users/extend/database_types.md]:
```suggestion
ddev can currently support most versions of MariaDB and MySQL database servers. You can configure the type that you want using the `.ddev/config.yaml` file, for example: 
```

File: docs/users/extend/database_types.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
 
-`mariadb_version: 10.4`
-`mysql_version: 8.0`
+`mariadb_version: 10.4` or `mysql_version: 8.0`
```
Comment (rfay) [inline docs/users/extend/database_types.md]:
```suggestion
(`mariadb_version` and `mysql_version` are mutually exclusive.)
```

**Comment by rfay** [comment]
Thanks for this @mglaman - If you'll either give me permissions on your fork (click the checkbox to allow maintainers to commit) or commit the changes suggested, we'll have it done.

**Comment by mglaman** [comment]
@rfay the PR has permission for you to push/force push/modify my branch for this PR

**Comment by rfay** [comment]
Thanks so much @mglaman

**Pull request merged**